conf/application.conf

```diff
@@ -111,6 +111,8 @@ microservice {
     }
 }
 
+contact-frontend.serviceId = "pension-scheme-return-sipp-frontend"
```

Comment:
Normally we provide serviceId in line 107 in contact-frontend (It is a required one) but also needs to be outside of it... Little interesting...